crypto/fipsmodule/aes/asm/aesni-x86.pl

```diff
@@ -153,17 +153,19 @@ sub aesni_generate1	# fully unrolled loop
 	&cmp		($rounds,11);
 	&jb		(&label("${p}128"));
 	&lea		($key,&DWP(0x20,$key));
-	&je		(&label("${p}192"));
-	&lea		($key,&DWP(0x20,$key));
```

Comment:
Why remove the `lea` here? (I remember that I made a similar change to this before where I removed an extra instruction and it actually resulted in a failure; the failure wasn't caught on travis CI because Travis CI's configuration caused the CPUID-based dispatching logic to choose a different code path when running the tests.)